packages/panzer/disc-fe/src/Panzer_IntegrationValues2.cpp

```diff
@@ -976,8 +976,13 @@ getJacobian(const bool cache,
   auto s_jac        = Kokkos::subdynrankview(aux.get_view(),       cell_range,Kokkos::ALL(),Kokkos::ALL(),Kokkos::ALL());
 
   // FIXME: Intrepid requires some specific types, so we re-wrap our views in non-const views
-  UnmanagedDynRankView<Scalar> nonconst_node_coord(const_cast<Scalar*>(s_node_coord.data()), num_evaluate_cells_,num_nodes,num_space_dim);
-  UnmanagedDynRankView<Scalar> nonconst_ref_coord(const_cast<Scalar*>(s_ref_coord.data()), num_evaluate_cells_,num_ip,num_space_dim);
+  //UnmanagedDynRankView<Scalar> nonconst_node_coord(const_cast<Scalar*>(s_node_coord.data()), num_evaluate_cells_,num_nodes,num_space_dim);
+  Kokkos::DynRankView<Scalar,PHX::Device> nonconst_node_coord("scratch: nonconst_node_coord", num_evaluate_cells_,num_nodes,num_space_dim);
+  Kokkos::deep_copy(nonconst_node_coord,s_node_coord);
```

Comment:
@rppawlo Ok, I've tried this a few different ways, but CUDA fails on all my attempts to wrap a non-const view around a const view. How do you build a unmanaged dynamic rank view? The version I pulled from empire (which agrees with PHX and Panzer) doesn't seem to work (it compiles and passes on everything except CUDA and CUDA-UVM-off). Is there something special I need to do with the pointer I pass to the view constructor?